test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -490,57 +505,57 @@ TEST_F(ClusterManagerImplTest, ValidClusterName) {
 }
 
 TEST_F(ClusterManagerImplTest, OriginalDstLbRestriction) {
-  const std::string json = R"EOF(
-  {
-    "clusters": [
-    {
-      "name": "cluster_1",
-      "connect_timeout_ms": 250,
-      "type": "original_dst",
-      "lb_type": "round_robin"
-    }]
-  }
+  const std::string yaml = R"EOF(
+static_resources:
+  clusters:
+  - name: cluster_1
+    connect_timeout: 0.250s
+    type: original_dst
+    lb_policy: round_robin
   )EOF";
 
   EXPECT_THROW_WITH_MESSAGE(
-      create(parseBootstrapFromJson(json)), EnvoyException,
+      create(parseBootstrapFromV2Yaml(yaml)), EnvoyException,
       "cluster: cluster type 'original_dst' may only be used with LB type 'original_dst_lb'");
 }
 
 TEST_F(ClusterManagerImplTest, OriginalDstLbRestriction2) {
-  const std::string json = R"EOF(
-  {
-    "clusters": [
-    {
-      "name": "cluster_1",
-      "connect_timeout_ms": 250,
-      "type": "static",
-      "lb_type": "original_dst_lb",
-      "hosts": [{"url": "tcp://127.0.0.1:11001"}]
-    }]
-  }
+  const std::string yaml = R"EOF(
+static_resources:
+  clusters:
+  - name: cluster_1
+    connect_timeout: 0.250s
+    type: static
+    lb_policy: original_dst_lb
+    hosts:
+    - socket_address:
+        address: 127.0.0.1
+        port_value: 11001
   )EOF";
 
   EXPECT_THROW_WITH_MESSAGE(
-      create(parseBootstrapFromJson(json)), EnvoyException,
+      create(parseBootstrapFromV2Yaml(yaml)), EnvoyException,
       "cluster: LB type 'original_dst_lb' may only be used with cluster type 'original_dst'");
 }
 
 TEST_F(ClusterManagerImplTest, SubsetLoadBalancerInitialization) {
-  const std::string json = R"EOF(
-  {
-    "clusters": [
-    {
-      "name": "cluster_1",
-      "connect_timeout_ms": 250,
-      "type": "static",
-      "lb_type": "round_robin",
-      "hosts": [{"url": "tcp://127.0.0.1:8000"}, {"url": "tcp://127.0.0.1:8001"}]
-    }]
-  }
+  const std::string yaml = R"EOF(
+static_resources:
+  clusters:
+  - name: cluster_1
+    connect_timeout: 0.250s
+    type: static
+    lb_policy: round_robin
+    hosts:
```

Comment:
Should we directly convert `hosts` to `load_assignment` here? (since `hosts` is deprecated).

Reply:
Done. I did this for _most_ cases. There's a few spots where v1-consuming functions are still used (i.e. `configDump` used in `ClusterManagerImplTest.InitializeOrder`) but I'd prefer to handle those in a follow-up PR given the LoC already in this PR.